application/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -69,66 +74,71 @@ $$$$
       apiConfig: ApiConfig,
       dbConfig: DatabaseConfig
   ) =
-    for {
-      connectionEc  <- ExecutionContexts.fixedThreadPool[IO](2)
-      transactionEc <- ExecutionContexts.cachedThreadPool[IO]
-      xa <- HikariTransactor.newHikariTransactor[IO](
-        "org.postgresql.Driver",
-        dbConfig.jdbcUrl,
-        dbConfig.dbUser,
-        dbConfig.dbPass,
-        connectionEc,
-        Blocker.liftExecutionContext(transactionEc)
-      )
-      collectionItemEndpoints = new CollectionItemEndpoints[IO](
-        apiConfig.defaultLimit,
-        apiConfig.enableTransactions,
-        apiConfig.enableTiles
-      )
-      collectionEndpoints = new CollectionEndpoints[IO](
-        apiConfig.enableTransactions,
-        apiConfig.enableTiles
-      )
-      allEndpoints = collectionEndpoints.endpoints ++ collectionItemEndpoints.endpoints ++ new SearchEndpoints[
-        IO
-      ].endpoints ++ new TileEndpoints[
-        IO
-      ](
-        apiConfig.enableTiles
-      ).endpoints
-      docs      = allEndpoints.toOpenAPI("Franklin", "0.0.1")
-      docRoutes = new SwaggerHttp4s(docs.toYaml, "open-api", "spec.yaml").routes[IO]
-      searchRoutes = new SearchService[IO](
-        apiConfig.apiHost,
-        apiConfig.defaultLimit,
-        apiConfig.enableTiles,
-        xa
-      ).routes
-      tileRoutes = new TileService[IO](apiConfig.apiHost, apiConfig.enableTiles, xa).routes
-      collectionRoutes = new CollectionsService[IO](xa, apiConfig).routes <+> new CollectionItemsService[
-        IO
-      ](
-        xa,
-        apiConfig
-      ).routes
-      router = CORS(
-        Router(
-          "/" -> ResponseLogger.httpRoutes(false, false)(
-            collectionRoutes <+> searchRoutes <+> tileRoutes <+> docRoutes
-          )
+    AsyncHttpClientCatsBackend.resource[IO]() flatMap { implicit backend =>
```

Comment:
Why not to pass it explicitly? It is a pretty heavy resources that can be much more clear to handle manually. 

As an argument towards making in explicit: STTP deprecates its methods where backend could be passed implicitly: https://github.com/softwaremill/sttp/blob/430328fdd3bc66a55d7169b7e2f27bbabbd1bcb3/core/src/main/scala/sttp/client3/RequestT.scala#L280